madebr/librwgta
# add cmake support

- fix:
  - file modes
  - trailing semicolons
  - commas at end of enumerator lists
  - IIItest
- add cmake support:
  - The option `LIBRW_PLATFORM` can be defined in 2 ways:
    - using librwgta as a subproject after librw (for which `LIBRW_PLATFORM` should have been defined already)
    - use the installed librw library (consume the `librw` target, exported by `librw`)
  - installing librwgta will export the `librwgta` target so consumers of librwgta can use it.
- add travis and appveyor CI
  - appveyor has artifacts

ISSUES:
- ps2 does not compile
- ‘char* strncat(char*, const char*, size_t)’ specified bound 1024 equals destination size [-Wstringop-overflow=]
- unused parameter 'xxx' [-Wunused-parameter]
- ...
